spk/autossh/Makefile

```diff
@@ -0,0 +1,17 @@
+SPK_NAME = autossh
+SPK_VERS = 1.4g
+SPK_REV = 1
+CHANGELOG = "Version 1.4g most notably addresses a bug that could see an alarm signal occur without a handler, causing autossh to exit inappropriately."
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = chrisellsworth
+DESCRIPTION = autossh is a program to start a copy of ssh and monitor it, restarting it as necessary should it die or stop passing traffic. The idea is from rstunnel (Reliable SSH Tunnel), but implemented in C.
+DISPLAY_NAME = autossh
+STARTABLE = no
+
+HOMEPAGE = https://www.harding.motd.ca/autossh/
+
+SPK_LINKS = /usr/local/bin/autossh:bin/autossh
```

Comment:
This can be simplyfied by `SPK_COMMANDS = bin/autossh`
`SPK_LINKS` is required for links with different name than the target.

Reply:
Thanks! Updated to use `SPK_LINKS` and verified this links correctly for me.